### PR TITLE
Add naughty strings

### PR DESCRIPTION
**Pull request by Cantido**
This change adds a generator for strings taken from the [Big List of Naughty Strings](https://github.com/minimaxir/big-list-of-naughty-strings).

I've added:

- [x] ~USAGE.md docs if applicable~ (Not applicable, this change adds a function to the String module)
- [x] CHANGELOG.md

### CODE REVIEW

File: lib/faker/string.ex [hunk 1]
```diff
@@ -24,4 +26,525 @@ defmodule Faker.String do
...
+  Returns a random string taken from the [Big List of Naughty Strings](https://github.com/minimaxir/big-list-of-naughty-strings).
+
+  ## Examples
+
+      iex> Faker.String.naughty()
+      "̦H̬̤̗̤͝e͜ ̜̥̝̻͍̟́w̕h̖̯͓o̝͙̖͎̱̮ ҉̺̙̞̟͈W̷̼̭a̺̪͍į͈͕̭͙̯̜t̶̼̮s̘͙͖̕ ̠̫̠B̻͍͙͉̳ͅe̵h̵̬͇̫͙i̹͓̳̳̮͎̫̕n͟d̴̪̜̖ ̰͉̩͇͙̲͞ͅT͖̼͓̪͢h͏͓̮̻e̬̝̟ͅ ̤̹̝W͙̞̝͔͇͝ͅa͏͓͔̹̼̣l̴͔̰̤̟͔ḽ̫.͕"
+      iex> Faker.String.naughty()
+      "1#QNAN"
+      iex> Faker.String.naughty()
+      "Craig Cockburn, Software Specialist"
```
Comment (vbrazo) [inline lib/faker/string.ex]: Could you increment and add 2 more examples please?

File: lib/faker/string.ex [hunk 2]
```diff
@@ -24,4 +26,525 @@ defmodule Faker.String do
...
+    "👩🏽",
+    "👾 🙇 💁 🙅 🙆 🙋 🙎 🙍",
+    "🐵 🙈 🙉 🙊",
+    "❤️ 💔 💌 💕 💞 💓 💗 💖 💘 💝 💟 💜 💛 💚 💙",
+    "✋🏿 💪🏿 👐🏿 🙌🏿 👏🏿 🙏🏿",
+    "🚾 🆒 🆓 🆕 🆖 🆗 🆙 🏧",
+    "0️⃣ 1️⃣ 2️⃣ 3️⃣ 4️⃣ 5️⃣ 6️⃣ 7️⃣ 8️⃣ 9️⃣ 🔟",
+    "🇺🇸🇷🇺🇸 🇦🇫🇦🇲🇸",
+    "🇺🇸🇷🇺🇸🇦🇫🇦🇲",
+    "🇺🇸🇷🇺🇸🇦",
```
Comment (vbrazo) [inline lib/faker/string.ex]: I wonder if we want to store emojis as multibytes. Thoughts?

Comment (Cantido) [reply lib/faker/string.ex]: I'm on the fence about it. I like being able to see what characters the bytes actually are, but then again the entire point of a lot of these strings is that stuff messes up when trying to display them. Storing all the tricky stuff as multibytes would definitely get rid of any problems with displaying the source file. Do you have a preference either way?

Comment (vbrazo) [reply lib/faker/string.ex]: @igas @anthonator thoughts?

Comment (igas) [reply lib/faker/string.ex]: I personally don't have preference.

**Comment by Cantido** [comment]
Oops I was making another contribution but forgot I had this pull request open still. I'll revert that and prepare a different branch.

Is there anything else y'all need from me in order to merge this?

File: lib/faker/string.ex (hunk 2 above)
Comment (anthonator) [reply lib/faker/string.ex:206]: I'm going to assume we're good with the current implementation in order to move forward.

**Review by anthonator — APPROVED** [review]

**Comment by syfgkjasdkn** [comment]
I'm getting

```
== Compilation error in file lib/faker/string.ex ==
** (SyntaxError) lib/faker/string.ex:144:15: invalid bidirectional formatting character in string: \u202A. If you want to use such character, use it in its escaped \u202A form instead
    (elixir 1.13.0) lib/kernel/parallel_compiler.ex:346: anonymous fn/5 in Kernel.ParallelCompiler.spawn_workers/7
could not compile dependency :faker, "mix compile" failed. Errors may have been logged above. You can recompile this dependency with "mix deps.compile faker", update it with "mix deps.update faker" or clean it with "mix deps.clean faker"
```

on elixir 1.13 for a string added in this PR. Is there any additional setup needed after this PR?

**Pull request merged**